spec/config.schema.json

```diff
@@ -1915,69 +1915,27 @@
                 "sampling": {
                   "type": "object",
                   "propertyNames": {
-                    "enum": ["type", "value", "server_url"]
+                    "enum": ["server_url", "trace_id_ratio"]
                   },
                   "allOf": [
```

Comment:
The `allOf` can also be removed because there is just one element in the array now.